sidecar-ffi/src/lib.rs

```diff
@@ -709,3 +710,18 @@ pub extern "C" fn ddog_sidecar_reconnect(
 ) {
     transport.reconnect(|| unsafe { factory() });
 }
+
+/// Return the path of the crashtracker unix domain socket.
+#[no_mangle]
+#[allow(clippy::missing_safety_doc)]
+pub unsafe extern "C" fn ddog_sidecar_get_crashtracker_unix_socket_path() -> ffi::CharSlice<'static>
```

Comment:
Who is the owner of this string, and is expected to free it?

Reply:
The caller. I mean, it returns libc::malloc, so the caller can just free() it.